rfc6962-bis.xml

```diff
@@ -1359,7 +1359,14 @@ but it is expected there will be a variety.
       </section>
       <section title="Auditing">
         <t>
-          Auditing is taking partial information about a log as input and verifying that this information is consistent with other partial information held. All clients described above may perform auditing as an additional function. The action taken by the client if audit fails is not specified, but note that in general if audit fails, the client is in possession of signed proof of the log's misbehavior.
+          Auditing ensures that the current published state of a log is reachable from previously published
+          states that are known to be good.  Audits are performed by monitors or TLS clients.
```

Comment:
How about:  "Auditing ensures that the current published state of a log is reachable from previously published states that are known to be good, and that the promises made by the log in the form of SCTs have been kept.  Audits are performed by monitors or TLS clients."

It's more verbose but provides specifics.  The original objection was about using the term 'partial information' which was too woolly.